leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -339,7 +353,7 @@ var ElementInspector = Backbone.View.extend({
         funcSegments.forEach(
             element => {
```

Comment:
We could change `element ` to `functionSegment ` here

Reply:
Nice will do in my next commit